modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -63,34 +65,10 @@ public void testSendNotification() throws Exception {
 			notificationQueueEntryLocalService.
 				getNotificationQueueEntriesCount());
 
-		NotificationTemplate notificationTemplate =
-			notificationTemplateLocalService.addNotificationTemplate(
-				NotificationTemplateUtil.createNotificationContext(
-					TestPropsValues.getUser(),
-					ListUtil.toString(getTermNames(), StringPool.BLANK),
-					RandomTestUtil.randomString(),
-					Arrays.asList(
-						createNotificationRecipientSetting(
-							"bcc",
-							"[%CURRENT_USER_EMAIL_ADDRESS%],bcc@liferay.com"),
-						createNotificationRecipientSetting(
-							"cc",
-							"[%CURRENT_USER_EMAIL_ADDRESS%],cc@liferay.com"),
-						createNotificationRecipientSetting(
-							"from", "[%CURRENT_USER_EMAIL_ADDRESS%]"),
-						createNotificationRecipientSetting(
-							"fromName",
-							Collections.singletonMap(
-								LocaleUtil.US, "[%CURRENT_USER_FIRST_NAME%]")),
-						createNotificationRecipientSetting(
-							"to",
-							Collections.singletonMap(
-								LocaleUtil.US,
-								"[%CURRENT_USER_EMAIL_ADDRESS%]"))),
-					ListUtil.toString(getTermNames(), StringPool.BLANK),
```

Comment:
I removed the `[%CURRENT_USER_EMAIL_ADDRESS%]` because we have a bug opened related to this. So we can fix the bug and add it here again in the same PR.

What do you think?

Reply:
For me ok, but we need to put this comment in the related ticket to register that is necessary come back the `[%CURRENT_USER_EMAIL_ADDRESS%]` to the test.

Reply:
Agree.